docs/ProductSpecifications.md

```diff
@@ -1,68 +1,63 @@
-# Product Specifications
+📢 Use this project, [contribute](https://github.com/vtex-apps/store-components) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion). 
 
-## Description
+# Product Specifications
 
 `ProductSpecifications` is a VTEX component that shows the technical specifications of a product.
 This component is meant to be used inside the `store.product` block and consume data from the `product-context`.
```

Comment:
I'd delete this sentence from here and put it in the configuration section. What do you thinj?